src/main/java/org/folio/circulation/domain/notice/TemplateContextUtil.java

```diff
@@ -414,13 +417,20 @@ public UserContext withAddressProperties(JsonObject address) {
           .with(UserContext.CITY, address.getString("city", null))
           .with(UserContext.REGION, address.getString("region", null))
           .with(UserContext.POSTAL_CODE, address.getString("postalCode", null))
-          .with(UserContext.COUNTRY_ID, address.getString("countryId", null))
+          .with(UserContext.COUNTRY_ID, getCountryNameByCountryId(address.getString(COUNTRY_ID, null)))
           .with(UserContext.ADDRESS_TYPE_NAME, address.getString("addressTypeName", null));
       } else {
         return this;
       }
     }
 
+    private String getCountryNameByCountryId(String countryId) {
+      if(StringUtils.isNotEmpty(countryId)){
+        return CountryCode.getByCodeIgnoreCase(countryId).getName();
```

Comment:
Might be good to throw an exception here if the 2 char code wasn't a valid code since I think we're not controlling where the code comes from.

Reply:
done